bin/prompt-command.js

```diff
@@ -105,6 +105,10 @@ module.exports = function promptForInstallation(packages, ...args) {
 			}
 		});
 	} else {
-		require(pathForCmd).default(...args); // eslint-disable-line
+          let p = require(pathForCmd), c = p[packages];
```

Comment:
Can you try to make this a bit prettier with better variable names?

Reply:
@ev1stensberg Hi Even, you might try refining the name as wish, I'd like to present this patch for discussion as I'm quite certain about `packages` -- as the name literally suggested, it might have multiple `package` names? The other possible fix might be changing the `@webpack-cli/xxx` modules, e.g.

``` js
function serve() {
    blah(...)
    blah(...)
}

export abc;
export xyz;
export default serve;
```

as `webpack-cli` seems to expect any `@webpack-cli/xxx` to offer a default export as a `function`, which is rational. But `webpack-cli` deserve a type check.

Reply:
Yes in theory the package that we want to load should export a default function and I think bombing an error is fine, in order to dictate a standard. Your pseudo solution looks fine